src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -51,6 +53,13 @@ public ApplyClause(IList<TransformationNode> transformations)
 
                     break;
                 }
+                else if (transformations[i].Kind == TransformationNodeKind.Compute)
+                {
+                    if (lastAggregateExpressions == null)

```

Comment:
>lastAggregateExpressions [](start = 24, length = 24)

`lastComputeExpressions`?

Reply:
No, lastAggregateExpressions to support scenario from ReferenceComputeAliasCreatedBeforeAggrageteThrows